cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CountryConversion.java

```diff
@@ -71,7 +70,23 @@ private CountryConversion() {
         }
     }
 
+    public static Optional<Country> fromIsoCode(String iso) {
+        if (iso == null) {
+            return Optional.empty();
+        }
+        try {
+            return Optional.of(Country.valueOf(iso.trim().toUpperCase()));
+        } catch (IllegalArgumentException ignored) {
+            // Ignore
+        }
+        return Optional.empty();
+    }
+
     public static Country defaultCountry() {
         return Country.values()[0];
     }
+
+    public static Country defaultCountryForBoundary() {
+        return Country.BD;
```

Comment:
Could you add a log to indicate that this method is called? I think it’s important to know that we invent information